evm_arithmetization/Cargo.toml

```diff
@@ -55,7 +57,7 @@ ripemd = { workspace = true }
 
 [features]
 default = ["parallel"]
-asmtools = ["hex"]
+#asmtools = ["hex"]
```

Comment:
why?

Reply:
workaround for something earlier in the process. Gone now.